draft-ietf-taps-impl.md

```diff
@@ -489,7 +489,7 @@ Instead, calling the ConnectionReceived event may be delayed until the first Mes
 
 ## Handling racing with "unconnected" protocols {#unconnected-racing}
 
-While protocols that use an explicit handshake to validate a Connection to a peer can be used for racing multiple establishment attempts in parallel, "unconnected" protocols such as raw UDP do not offer a way to validate the presence of a peer or the usability of a Connection without application feedback. An implementation should consider such a protocol stack to be established as soon as a local route to the peer endpoint is confirmed.
+While protocols that use an explicit handshake to validate a Connection to a peer can be used for racing multiple establishment attempts in parallel, "unconnected" protocols such as raw UDP do not offer a way to validate the presence of a peer or the usability of a Connection without application feedback. An implementation should consider such a protocol stack to be established as soon as connectivity with the peer endpoint is established.
 
 However, if a peer is not reachable over the network using the unconnected protocol, or data cannot be exchanged for any other reason, the application may want to attempt using another candidate Protocol Stack. The implementation should maintain the list of other candidate Protocol Stacks that were eligible to use.
```

Comment:
We can recommend that applications other than a "fire-and-forget" case make their connections stateful, such as with a Message Framer on top of UDP that can verify when the peer is reachable.